docs/content/en/docs/getting-started/optional/cni.md

```diff
@@ -60,8 +60,56 @@ Starting with release 0.8, the plugin should be specified using the new `cniConf
 > NOTE: EKS Anywhere allows specifying only 1 plugin for a cluster and does not allow switching the plugins
 after the cluster is created.
 
+### Helm Values Configuration for Cilium plugin
+
+Starting with recent releases, Cilium can be configured using Helm values directly through the `helmValues` field. This provides full flexibility to configure any Cilium Helm chart parameter.
+
+When `helmValues` is specified, it takes precedence over all other Cilium-specific fields in the configuration. The `helmValues` field accepts standard Cilium Helm chart values in YAML format.
+
+Example configuration using `helmValues`:
+
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster-name
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 192.168.0.0/16
+    services:
+      cidrBlocks:
+      - 10.96.0.0/12
+    cniConfig:
+      cilium:
+        helmValues:
+          policyEnforcementMode: always
+          egressMasqueradeInterfaces: eth0
+          routingMode: native
+          ipv4NativeRoutingCIDR: 192.168.0.0/16
+          cni:
+            exclusive: false
+          hubble:
+            enabled: true
+            relay:
+              enabled: true
+            ui:
+              enabled: true
+```
+
+For a complete list of available Helm values, refer to the official [Cilium Helm chart documentation](https://docs.cilium.io/en/stable/helm-reference/).
+
+{{% alert title="Note" color="primary" %}}
+When using `helmValues`, all legacy Cilium configuration fields (such as `policyEnforcementMode`, `egressMasqueradeInterfaces`, `routingMode`, `ipv4NativeRoutingCIDR`, `ipv6NativeRoutingCIDR`, and `cniExclusive`) are ignored. Configure these parameters within the `helmValues` field instead.
+{{% /alert %}}
+
 ### Policy Configuration options for Cilium plugin
 
+{{% alert title="Deprecated" color="warning" %}}
+The `policyEnforcementMode` field is deprecated. Please use `helmValues` instead. See the [Helm Values Configuration](#helm-values-configuration-for-cilium-plugin) section above.
```

Comment:
```suggestion
The `policyEnforcementMode` field is deprecated. Please use `helmValues` instead. See the [Helm Values Configuration](#helm-values-configuration-for-cilium-plugin) section for details.
```
Sections have a way of moving around, so it's good to avoid things like "above" or "below" if you can.